spec/definitions.yaml

```diff
@@ -776,4 +776,11 @@ CustomProperty:
         example: ["Value 1", "Value 2"]
   required:
     - name
-
+FeatureFlagCopyObject:
+  type: object
+  properties:
+    key:
+      type: string
+    version:
+      type: integer
+      example: 65
```

Comment:
Neither of these is optional, right? In that case I think you want to add this:
```
  required:
    - key
    - version
```

Reply:
ahh great catch thanks! So key is required, but version is optional.

Reply:
Unfortunately I don't know the answer to the build error you're seeing.